client/src/app/s-degree-plan-editor/s-degree-plan-editor.component.html

```diff
@@ -230,28 +244,34 @@
         </div>
         <div class="add-year-card">
           <button
-            *ngIf="plan.isAddingYear === false"
+            *ngIf="addYearWidget.active === false"
```

Comment:
change to ngIf="!addYearWidget.active" 
! is the same as false => if (not true) = if(false)

Reply:
Changed